jdaviz/configs/imviz/imviz.yaml

```
@@ -19,6 +19,8 @@ toolbar:
   - g-coords-info
 tray:
   - g-metadata-viewer
+  - g-export-plot
+  - g-plot-options
```

At least for Imviz, I propose rearranging the order to reflect better typical workflow logic. You want to adjust display first but save the plot last.

```suggestion
  - g-plot-options
  - g-metadata-viewer
```

---

As per offline convo, we can defer this.